### ISSUE TITLE
Create action to run DRC checks on GDS

### ISSUE THREAD

**Issue by mithro** [google/skywater-pdk-actions#5]

**Comment by mithro** [comment]
@donn has a WIP example at https://github.com/google/skywater-pdk-libs-sky130_fd_sc_hd/pull/6

**Comment by mithro** [comment]
@donn -- This should be done now, right?

**Comment by donn** [comment]
Yes, though this only runs DRC with Magic. Which is fine by me but I'm wondering if some other members in the team would also prefer DRC with Klayout.
